### DISCUSSION TITLE
Possible to run `bun test` with vite / svelte setup?

### DISCUSSION THREAD

**Discussion by kjk** [oven-sh/bun#12402] [Q&A]
I have a `Svelte` app that I build with `vite` which involves `vite` transpiling which is how `.svelte` files are converted to `.js` or how not-really-javascript like `import myFunctionsRaw from "./note-custom-functions.md?raw";` are implemented.

I cannot write bun test that import `.svelte` files or files that use the above non-standard `import`.

Is there a way to make it work?

I've read `bun test` docs but there doesn't seem anything related to that.

It seems to make this work when `bun test` does `import x from "./foo.svelte` or `import y from "./foo.js"`, those files would have to be first pre-processed by my existing vite setup.

Is that possible? How to configure `bun` (or maybe `vite`?) to make that work?

**Comment by FlorianB-DE** [comment]
Any Progress?
